test_app/integration_tests/mocked/video_presentation_service_test.dart

```diff
@@ -5,91 +5,93 @@ import 'package:dolbyio_comms_sdk_flutter/dolbyio_comms_sdk_flutter.dart';
-  });
-
-  testWidgets('VideoPresentationService: seek', (tester) async {
-    await dolbyioCommsSdkFlutterPlugin.videoPresentation.seek(1);
-    await expectNative(
-        methodChannel: videoPresentationServiceAssertsMethodChannel,
-        assertLabel: "assertSeekArgs",
-        expected: {"hasRun": true, "timestamp": 1});
-  });
+    testWidgets('VideoPresentationService: stop', (tester) async {
+      await dolbyioCommsSdkFlutterPlugin.videoPresentation.stop();
+      await expectNative(
+          methodChannel: videoPresentationServiceAssertsMethodChannel,
+          assertLabel: "assertStopArgs",
+          expected: {"hasRun": true});
+    });
 
-  testWidgets('VideoPresentationService: state', (tester) async {
-    await dolbyioCommsSdkFlutterPlugin.videoPresentation.play();
-    var statePlay =
-        await dolbyioCommsSdkFlutterPlugin.videoPresentation.state();
-    expect(statePlay, VideoPresentationState.play);
+    testWidgets('VideoPresentationService: play', (tester) async {
+      await dolbyioCommsSdkFlutterPlugin.videoPresentation.play();
+      await expectNative(
+          methodChannel: videoPresentationServiceAssertsMethodChannel,
+          assertLabel: "assertPlayArgs",
+          expected: {"hasRun": true});
+    });
 
-    await expectNative(
-        methodChannel: videoPresentationServiceAssertsMethodChannel,
-        assertLabel: "assertStateArgs",
-        expected: {"hasRun": true, "state": 1});
+    testWidgets('VideoPresentationService: pause', (tester) async {
+      await dolbyioCommsSdkFlutterPlugin.videoPresentation.pause(1);
+      await expectNative(
+          methodChannel: videoPresentationServiceAssertsMethodChannel,
+          assertLabel: "assertPauseArgs",
+          expected: {"hasRun": true, "timestamp": 1});
+    });
 
-    await resetSDK();
+    testWidgets('VideoPresentationService: seek', (tester) async {
+      await dolbyioCommsSdkFlutterPlugin.videoPresentation.seek(1);
+      await expectNative(
+          methodChannel: videoPresentationServiceAssertsMethodChannel,
+          assertLabel: "assertSeekArgs",
+          expected: {"hasRun": true, "timestamp": 1});
+    });
 
-    if (!Platform.isAndroid) {
-      await dolbyioCommsSdkFlutterPlugin.videoPresentation.stop();
-      var stateStop =
-      await dolbyioCommsSdkFlutterPlugin.videoPresentation.state();
-      expect(stateStop, VideoPresentationState.stopped);
+    testWidgets('VideoPresentationService: state', (tester) async {
+      await dolbyioCommsSdkFlutterPlugin.videoPresentation.play();
+      var statePlay =
+          await dolbyioCommsSdkFlutterPlugin.videoPresentation.state();
+      expect(statePlay, VideoPresentationState.play);
 
       await expectNative(
           methodChannel: videoPresentationServiceAssertsMethodChannel,
           assertLabel: "assertStateArgs",
-          expected: {"hasRun": true, "state": 0});
+          expected: {"hasRun": true, "state": 1});
```

Comment:
Why expected state was change from 0 to 1 ?

Reply:
Oh sorry, I didn't change the state. The code is rearranged I suppose. For the play() method the state was 1 and for the stop() method the state was 0.
The state you’re looking for is in the below part of the code I think.

Reply:
ok, my mistake. Sometimes git show changes in very weired way :)

Reply:
That's true, I agree with that :)